amy/consents/migrations/0002_term_help_text.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2021-04-05 19:56
```

Comment:
I think you have another `0002` migration in #1880.

Reply:
Good catch. Thanks!